### ISSUE TITLE
Fixing Google AIP api linter

### ISSUE THREAD

**Issue by glimchb** [opiproject/opi-api#195]
```
docker run --user=$(id -u):$(id -g) --rm -v "opiproject/opi-api/storage"/v1alpha1/:/out -w /out --entrypoint=sh ghcr.io/docker-multiarch/google-api-linter:1.36.0 -c "api-linter /out/*.proto --output-format summary"
+--------------------------------------+------------------+----------------+
|                 RULE                 | TOTAL VIOLATIONS | VIOLATED FILES |
+--------------------------------------+------------------+----------------+
| core::0191::file-layout              |                1 |              1 |
| core::0140::reserved-words           |                1 |              1 |
| core::0191::java-outer-classname     |                1 |              1 |
| core::0142::time-field-type          |                2 |              1 |
| core::0203::optional                 |                2 |              1 |
| core::0126::unspecified              |                3 |              1 |
| core::0141::count-suffix             |                4 |              4 |
| core::0123::resource-annotation      |                4 |              4 |
| core::0192::only-leading-comments    |                6 |              6 |
| core::0136::http-method              |               12 |              3 |
| core::0136::http-body                |               12 |              3 |
| core::0127::resource-name-extraction |               12 |              3 |
| core::0141::forbidden-types          |               14 |              2 |
| core::0127::http-annotation          |               24 |              6 |
| core::0136::http-uri-suffix          |               30 |              3 |
| core::0192::has-comments             |              282 |              6 |
+--------------------------------------+------------------+----------------+
Linted 13 proto files
```

**Comment by glimchb** [comment]
with latest merged code https://github.com/opiproject/opi-api/blob/main/storage/Makefile#L18
```
	docker run --user=$$(id -u):$$(id -g) --rm --entrypoint=sh -v "${PWD}/../common/v1":/common -v "${PWD}"/v1alpha1/:/out -w /out ghcr.io/docker-multiarch/google-api-linter:1.36.0 -c "api-linter -I /common /out/*.proto --output-format github --disable-rule=core::0192::has-comments --disable-rule=core::0136 --disable-rule=core::0127 --set-exit-status"
```
still ignoring rules that need to enable them:
- `core::0192::has-comments`
- `core::0136`
- `core::0127`

**Comment by glimchb** [comment]
after #202 
```
+--------------------------------------------+------------------+----------------+
|                    RULE                    | TOTAL VIOLATIONS | VIOLATED FILES |
+--------------------------------------------+------------------+----------------+
| core::0133::http-body                      |                3 |              2 |
| core::0132::request-unknown-fields         |                4 |              2 |
| core::0135::request-name-required          |                6 |              3 |
| core::0133::request-resource-behavior      |                6 |              3 |
| core::0134::request-unknown-fields         |                6 |              3 |
| core::0134::request-mask-required          |                6 |              3 |
| core::0133::request-parent-required        |                6 |              3 |
| core::0134::method-signature               |                6 |              3 |
| core::0134::http-body                      |                6 |              3 |
| core::0158::request-page-size-field        |                6 |              3 |
| core::0131::method-signature               |                6 |              3 |
| core::0131::request-name-required          |                6 |              3 |
| core::0134::request-resource-field         |                6 |              3 |
| core::0133::request-resource-field         |                6 |              3 |
| core::0131::http-uri-name                  |                6 |              3 |
| core::0133::http-uri-parent                |                6 |              3 |
| core::0132::request-parent-required        |                6 |              3 |
| core::0158::request-page-token-field       |                6 |              3 |
| core::0158::response-next-page-token-field |                6 |              3 |
| core::0135::method-signature               |                6 |              3 |
| core::0158::response-plural-first-field    |                6 |              3 |
| core::0134::http-uri-name                  |                6 |              3 |
| core::0132::response-unknown-fields        |                6 |              3 |
| core::0133::method-signature               |                6 |              3 |
| core::0135::http-uri-name                  |                6 |              3 |
| core::0131::request-unknown-fields         |                7 |              3 |
| core::0135::request-unknown-fields         |                7 |              3 |
| core::0127::resource-name-extraction       |               12 |              3 |
| core::0127::http-annotation                |               24 |              6 |
| core::0192::has-comments                   |              281 |              6 |
+--------------------------------------------+------------------+----------------+
```

**Comment by glimchb** [comment]
`0158` fixed by #203

**Comment by glimchb** [comment]
Fixed by #218 :

- [storage] [lint] Re-enable rule=core::0135
- [storage] [lint] Re-enable rule=core::0131
- [storage] [lint] Re-enable rule=core::0132
- [storage] [lint] Re-enable rule=core::0134
- [storage] [lint] Re-enable rule=core::0133

**Comment by glimchb** [comment]
Fixed by #317:
Now left:
```
+--------------------------+------------------+----------------+
|           RULE           | TOTAL VIOLATIONS | VIOLATED FILES |
+--------------------------+------------------+----------------+
| core::0192::has-comments |              408 |              9 |
+--------------------------+------------------+----------------+
```

**Comment by glimchb** [comment]
Fixed by https://github.com/opiproject/opi-api/pull/346:
Now left:
```
+--------------------------+------------------+----------------+
|           RULE           | TOTAL VIOLATIONS | VIOLATED FILES |
+--------------------------+------------------+----------------+
| core::0192::has-comments |              272 |              9 |
+--------------------------+------------------+----------------+
```
